app/static/src/app/components/run/RunModelPlot.vue

```diff
@@ -94,10 +97,22 @@ export default defineComponent({
         });
 
         watch(solution, () => {
-            baseData.value = solution.value(0, 100, nPoints); // TODO: default end time will eventually be a prop
+            // TODO: default end time will eventually be configured in the app
+            baseData.value = solution.value(0, 100, nPoints);
+
             drawPlot();
         });
 
+        onMounted(() => {
```

Comment:
on Mounted now needed to redraw the model when switch tab back from another, and onUnmounted to disconnect the resizeObserver when the user closes the tab. The model running will probably be moved out of the plot component when we have fuller functionality.